watertap/unit_models/electrodialysis_1D.py

```diff
@@ -1556,6 +1690,170 @@ def eq_dl_thickness(self, mem, side, t, x):
                     * self.current_dens_lim_x[t, x] ** -1
                 )
 
+    def _get_fluid_dimensionless_quantities(self):
+        self.general_mass_diffusivity = Var(
+            initialize=1e-9,
+            bounds=(1e-16, 1e-6),
+            units=pyunits.meter**2 * pyunits.second**-1,
+            doc="The mass diffusivity of the solute as molecules (not individual ions)",
+        )
+        self.hydraulic_diameter = Var(initialize=1e-3, units=pyunits.meter)
+        self.Re = Var(
+            initialize=100, units=pyunits.dimensionless, doc="Reynolds Number"
+        )
+        self.Sc = Var(
+            initialize=2000, units=pyunits.dimensionless, doc="Schmidt Number"
+        )
+        self.Sh = Var(
+            initialize=100, units=pyunits.dimensionless, doc="Sherwood Number"
+        )
+
+        if self.config.hydraulic_diameter_method == HydraulicDiameterMethod.defined:
+            _log.warning("Do not forget to FIX the channel hydraulic diameter in [m]!")
+        else:
+
+            @self.Constraint(
+                doc="To calculate hydraulic diameter",
+            )
+            def eq_hydraulic_diameter(self):
+                if (
+                    self.config.hydraulic_diameter_method
+                    == HydraulicDiameterMethod.conventional
+                ):
+                    return (
+                        self.hydraulic_diameter
+                        == 2
+                        * self.channel_height
+                        * self.cell_width
+                        * self.spacer_porosity
+                        * (self.channel_height + self.cell_width) ** -1
+                    )
+                else:
+                    self.spacer_specific_area = Var(
+                        initialize=1e4, units=pyunits.meter**-1
+                    )
+                    return (
+                        self.hydraulic_diameter
+                        == 4
+                        * self.spacer_porosity
+                        * (
+                            2 * self.channel_height**-1
+                            + (1 - self.spacer_porosity) * self.spacer_specific_area
+                        )
+                        ** -1
+                    )
+
+        @self.Constraint(
+            doc="To calculate Re",
+        )
+        def eq_Re(self):
+
+            return (
+                self.Re
```

Comment:
```suggestion
                self.N_Re
```
For dimensionless numbers, the IDAES standard name should start with `N_`